taxi/models.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return str(self.name)
```

Comment:
I guess this `str()` call is not necessary, since `CharField` value should already be a string.